assets/scss/_custom.scss

```diff
@@ -401,8 +401,8 @@ body {
 }
 
 .deprecation-warning, .pageinfo.deprecation-warning {
-    padding: 20px;
-    margin: 20px 0;
+    padding: 10px;
+    margin: 10px 0;
```

Comment:
Since we're looking at this CSS, can we make it more responsive? I'd usually do this with something like
```css
    padding: calc(max(10px, FIXMEvmin));
    margin: calc(max(10px, FIXMEvw)) 0;
```
or perhaps
```css
    padding: calc(max(1.5ch, FIXMEvmin));
    margin: calc(max(10px, FIXMEvw)) 0;
```

`clamp()` can be useful too, per https://css-tricks.com/min-max-and-clamp-are-css-magic/

Swap in appropriate values for `FIXME` - possibly different for each occurrence. BTW you need a `calc()` to stop Sass doing its own thing with `max()`.

Reply:
ack, switched to clamp to range between 10px (looks good on mobile) and 20px (looks fine on desktop) based on vmin and vh